### PR TITLE
chore(proxy): add support for more syntaxes

### LINKED ISSUE #1258: Limited source code syntax highlighting
Currently only a limited amount of file extensions seem to be syntax highlighted when browsing source code. Apparent examples missing are common configuration formats often found at the top-level of repositories like: Makefile and TOML.

Files people expected to work:

* Typescript
* TOML
* tsx
* svelte
* solidity

### PR DESCRIPTION
**Pull request by cloudhead**
I'm not sure how this will be bundled, but it works locally with `yarn start` :thinking: 

---

Closes #1258

### CODE REVIEW

**Comment by cloudhead** [comment]
Also I think eventually we'll have to put a lot of work managing these, getting them to match with our theme, etc., and perhaps we'll want to move them to their own repo :man_shrugging:, but I think for now this should do.

**Comment by rudolfs** [comment]
> I'm not sure how this will be bundled, but it works locally with yarn start 🤔

https://github.com/radicle-dev/radicle-upstream/pull/1405/commits/e2f92ea2cb6076465b2e8b0a77a8b1c4a18a8fc7

**Comment by xla** [comment]
> Also I think eventually we'll have to put a lot of work managing these, getting them to match with our theme, etc., and perhaps we'll want to move them to their own repo man_shrugging, but I think for now this should do.

Can you please create follow-up issues for these points @cloudhead - so we don't forget.

**Review by xla — APPROVED** [review]
Tested and validated in dist builds on linux.

🏞 🎀 🍖 🖕

File: proxy/coco/src/source.rs [hunk 1]
```diff
@@ -50,7 +50,29 @@ lazy_static::lazy_static! {
...
+            let exe_path = std::env::current_exe().expect("Can't get current exe path");
+            let root_path = exe_path
+                .parent()
+                .expect("Could not get parent path of current executable");
+            let mut tmp = root_path.to_path_buf();
+            tmp.push("assets");
+            let asset_path = tmp.to_str().expect("Couldn't convert pathbuf to str");
+
+            // Don't crash if we aren't able to load additional syntaxes for some reason.
+            builder.add_from_folder(asset_path, true).ok();
```
Comment (geigerzaehler) [inline proxy/coco/src/source.rs:72]: Can we print a warning if this doesn’t work?

Comment (rudolfs) [reply proxy/coco/src/source.rs]: https://github.com/radicle-dev/radicle-upstream/pull/1405/commits/c705d8f764008cd90a56d6994830e8d113b72553

**Review by geigerzaehler — APPROVED** [review]

**Review by xla — DISMISSED** [review]
🎗 🐩 👛 🛠

**Pull request merged**